features/com.google.cloud.tools.eclipse.3rdparty.feature/feature.xml

```diff
@@ -78,7 +78,14 @@
          id="org.yaml.snakeyaml"
          download-size="0"
          install-size="0"
-         version="1.17"
+         version="1.17.0"
+         unpack="false"/>
+         
+ <plugin
+         id="org.dadacoalition.yedit"
```

Comment:
Should use the import-feature to pull in `org.dadacoaltion.yedit.feature`.

Reply:
What is the import-feature? How does that work?

Reply:
We can either use the _Included Features_ tab (which binds to a specific version, which is ok in our case), or the _Dependencies_ tab which allows importing a feature or plugin with a version range.

Reply:
It sounds like you're referring to something in the PDE UI. I'm not really using that. The question is what the code should say and what it means.

Reply:
I think what you're asking for is something like this:

```
  <requires>
    <import
         feature="org.dadacoalition.yedit.feature"
         version="0.0.0"/>
  </requires>
```

However, "Feature reference 'org.dadacoalition.yedit.feature' cannot be resolved	feature.xml	/com.google.cloud.tools.eclipse.3rdparty.feature	line 100"

Reply:
Or maybe this?

```
  <requires>
    <import plugin="org.dadacoalition.yedit"/>
  </requires>
```

Reply:
The _Included Features_ is:
```
   <includes
         id="org.dadacoalition.yedit.feature"
         version="0.0.0"/>
```